bazel/repositories.bzl

```diff
@@ -73,16 +73,22 @@ def _build_recipe_repository_impl(ctxt):
 
     # Run the build script.
     environment = {}
-    print("Fetching external dependencies...")
+    if ("ENVOY_BAZEL_VERBOSE" in ctxt.os.environ):
+        print("Fetching external dependencies...")
     result = ctxt.execute(
         ["./repositories.sh"] + ctxt.attr.recipes,
         environment = environment,
         quiet = False,
     )
-    print(result.stdout)
-    print(result.stderr)
-    print("External dep build exited with return code: %d" % result.return_code)
+    if ("ENVOY_BAZEL_VERBOSE" in ctxt.os.environ):
+        print(result.stdout)
+        print(result.stderr)
+        print("External dep build exited with return code: %d" % result.return_code)
     if result.return_code != 0:
+        if ("ENVOY_BAZEL_VERBOSE" not in ctxt.os.environ):
```

Comment:
I think we should have it verbose by default, since users want to understand what is going on while Bazel is sitting there doing the long build of external deps. It's fine adding an option to disable..